app/src/main/java/com/hammer/talkbbokki/presentation/comment/CommentsDestination.kt

```diff
@@ -0,0 +1,26 @@
+package com.hammer.talkbbokki.presentation.comment
+
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavType
+import androidx.navigation.compose.composable
+import androidx.navigation.navArgument
+import com.hammer.talkbbokki.presentation.navigation.TalkbbokkiNavigationDestination
+
+object CommentsDestination : TalkbbokkiNavigationDestination {
+    override val route: String
+        get() = "comments_route"
+}
+
+fun NavGraphBuilder.commentsGraph(
+    onBackClick: () -> Unit,
+    navigateToCommentDetail: () -> Unit
+) {
+    composable(route = CommentsDestination.route + "?topicId={topicId}",
```

Comment:
🚫 **[ktlint]** <standard:argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)